code/controllers/subsystem/minor_mapping.dm

```diff
@@ -1,5 +1,5 @@
 #define REGAL_RAT_CHANCE 2
-#define PLAGUE_RAT_CHANCE 5
+#define PLAGUE_RAT_CHANCE 0
```

Comment:
> Someone can fix it if they want, but currently having 4 rats running around on lowpop with a 100% chance to infect with a plague that medbay can't cure while hiding under tables and shit so they can't be killed is bad.

cure is spaceallin, most basic virus curing chem, after that you can't be infected anymore only if you get cloned, also skill issue you can alt click tables to hit under them

the number is up to change, and they shouldn't spawn roundstart anymore


```suggestion
#define PLAGUE_RAT_CHANCE 1
```